deployment.rb

```diff
@@ -91,6 +91,7 @@ def load_configuration
     'ruby_installer'              => rack_env == :development ? 'rbenv' : 'system',
     'root_dir'                    => root_dir,
     'section_projects'            => [:staging, :adhoc, :development].include?(rack_env),
+    'slack_logging'               => false,
```

Comment:
Nope, at least not yet. But I also didn't see where `hip_chat_logging` is set anywhere else.